frontend/app/assets/stylesheets/spree/frontend/variables/bootstrap-overrides.scss

```diff
@@ -1,3 +1,4 @@
+@import 'variables';
```

Comment:
I suggest changing the order of files 'bootsrap-overrides' and 'varaibles' in `frontend/app/assets/stylesheets/spree/frontend/application.scss` rather than import it twice